src/vision/calib/Calibration.cpp

```diff
@@ -0,0 +1,210 @@
+#include "MonoCalibrator.hpp"
+#include "wfov_camera_msgs/WFOVImage.h"
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/image_encodings.h>
+#include <string>
+#include <vector>
```

Comment:
Wanna sort these alphabetically and separate the standard library headers from robosub-specific headers by a newline?